starter-code/pizza.js

```diff
@@ -1 +1,93 @@
-// Write your Pizza Builder JavaScript in this file.
+$( document ).ready(function() {
```

Comment:
you don't have to put every function that you will use for this inside your `$(docuemnt).ready(function())`. Only what you want loaded when the page loads.